core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -38,10 +38,14 @@ oppia.directive('editorNavigation', [function() {
         $scope.postTutorialHelpPopoverIsShown = false;
 
         $scope.$on('openPostTutorialHelpPopover', function() {
-          $scope.postTutorialHelpPopoverIsShown = true;
-          $timeout(function() {
+          if ($scope.isLargeScreen) {
```

Comment:
@kevinlee12 `openPostTutorialHelpPopover` triggers when the user comes to create explorations for the first time.
If adopted this code:
```
   if ($scope.isLargeScreen) {
            $scope.$on('openPostTutorialHelpPopover', function() {
            $scope.postTutorialHelpPopoverIsShown = true;
            $timeout(function() {
              $scope.postTutorialHelpPopoverIsShown = false;
            }, 5000);
            });
          } else {
            $scope.postTutorialHelpPopoverIsShown = false;
          }
```
The helptool tip doesnt appear at all. Because, scripts of isLargeScreen execute first and misses **openPostTutorialHelpPopover** before it could initialize.